orbit/cmd/desktop/desktop.go

```diff
@@ -110,6 +111,19 @@ func main() {
 		log.Info().Msgf("got a TUF update root: %s", tufUpdateRoot)
 	}
 
+	if runtime.GOOS == "linux" {
+		// Ensure only one instance of Fleet Desktop is running at a time
+		lockFile, err := getLockfile()
+		if err != nil {
+			log.Fatal().Err(err).Msg("could not secure lock file")
```

Comment:
What happens if you want to restart a new Fleet Desktop instance because we pushed a new update?

Reply:
orbit should always kill fleet under normal circumstances through the flow you linked above, realistically this should never happen. This is just a last resort to prevent multiple tray icons